src/sentry/api/endpoints/organization_slugs.py

```diff
@@ -0,0 +1,75 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from django.db import transaction
+
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.models import Project
+from django.core.validators import validate_slug, ValidationError
+
+
+class SlugsUpdateEndpoint(OrganizationEndpoint):
```

Comment:
lets avoid documenting this as I'd prefer we remove it and just use a "bulk project update" endpoint instead